src/course-updates/update-form/UpdateForm.jsx

```diff
@@ -0,0 +1,140 @@
+      'update-form__inner-first': isFirstUpdate,
+    })}
+    >
+      <Formik
+        initialValues={courseUpdatesInitialValues}
+        validationSchema={validationSchema}
+        validateOnBlur
+        onSubmit={onSubmit}
+      >
+        {({
+          values, handleSubmit, isValid, setFieldValue,
+        }) => (
+          <>
+            <h3 className="update-form-title">{formTitle}</h3>
+            {(requestType !== REQUEST_TYPES.edit_handouts) && (
+              <Form.Group className="mb-4 datepicker-field datepicker-custom">
+                <Form.Control.Feedback className="datepicker-float-labels">
+                  {intl.formatMessage(messages.updateFormDate)}
+                </Form.Control.Feedback>
+                <div className="position-relative">
+                  <Icon
+                    src={CalendarIcon}
+                    className="datepicker-custom-control-icon"
+                    alt={intl.formatMessage(messages.updateFormCalendarAltText)}
+                  />
+                  <DatePicker
+                    name="date"
+                    data-testid="course-updates-datepicker"
+                    selected={convertToDateFromString(values.date)}
+                    dateFormat={DATE_FORMAT}
+                    className={classNames('datepicker-custom-control', {
+                      'datepicker-custom-control_isInvalid': !isValid,
+                    })}
+                    autoComplete="off"
+                    selectsStart
+                    showPopperArrow={false}
+                    onChange={(value) => {
+                      if (!isValidDate(value)) {
+                        return;
+                      }
+                      setFieldValue('date', convertToStringFromDate(value));
+                    }}
+                  />
+                </div>
+                {!isValid && (
+                  <div className="datepicker-field-error">
+                    <Icon src={ErrorIcon} alt={intl.formatMessage(messages.updateFormErrorAltText)} />
+                    <span className="message-error">{intl.formatMessage(messages.updateFormInValid)}</span>
+                  </div>
+                )}
+              </Form.Group>
+            )}
+            <Form.Group className="m-0 mb-3">
+              <WysiwygEditor
+                initialValue={currentContent}
+                data-testid="course-updates-wisiwyg-editor"
+                name={contentFieldName}
+                minHeight={300}
+                onChange={(value) => {
+                  setFieldValue(contentFieldName, value || DEFAULT_EMPTY_WYSIWYG_VALUE);
+                }}
+              />
+            </Form.Group>
+            <ActionRow>
+              <Button variant="tertiary" type="button" onClick={close}>
+                {intl.formatMessage(messages.cancelButton)}
+              </Button>
+              <Button variant="primary" onClick={handleSubmit} type="submit">
```

Comment:
By default, the Pragon Button component has variant="primary". Let`s try to delete this.

Reply:
Done